libnet/configure.ac

```diff
@@ -4,25 +4,50 @@ dnl All rights reserved.
 dnl
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT([libnet],[1.2-rc3])
+AC_INIT([libnet],[1.2-rc3],,,[http://github.com/sam-github/libnet])
 AC_MSG_RESULT(beginning autoconfiguration process for libnet-${PACKAGE_VERSION} ...)
 AC_CANONICAL_TARGET
 AC_CONFIG_SRCDIR([src/libnet_build_ip.c])
 AC_CONFIG_MACRO_DIR([m4])
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign subdir-objects no-dependencies -Wall])
 AC_CONFIG_HEADERS([include/config.h])
-AM_MAINTAINER_MODE
 
 dnl
 dnl Check for the usual programs
 dnl
 AC_PROG_CC
 AC_PROG_INSTALL
 AC_USE_SYSTEM_EXTENSIONS
+AX_CFLAGS_WARN_ALL
 dnl disable shared libraries (for now)
 dnl AC_DISABLE_SHARED
+AM_PROG_AR
 LT_INIT
 
+dnl
+dnl Check for DOxygen and enable its features
+dnl
+
+# See /usr/share/aclocal/ax_prog_doxygen.m4 and
+# http://www.bioinf.uni-freiburg.de/~mmann/HowTo/automake.html#doxygenSupport
+
+DX_DOXYGEN_FEATURE(ON)
+DX_DOT_FEATURE(ON)
+DX_HTML_FEATURE(ON)
+DX_CHM_FEATURE(OFF)
+DX_CHI_FEATURE(OFF)
+DX_MAN_FEATURE(ON)
+DX_RTF_FEATURE(OFF)
+DX_XML_FEATURE(OFF)
+DX_PDF_FEATURE(ON)
```

Comment:
Hey
Every doc type except HTML, man and RTF require additional packages.
IMO PDF requires way too many to be enabled by default.
Apart from graphviz, you also need TeX Live. On my Ubuntu, I tried the smallest TeX Live package available, `texlive-latex-base`. That wasn't enough. 
The next best, `texlive-latex-recommended`, didn't suffice either. 
Then came `texlive-latex-extra` , but it's not enough still. 
Last try was installing `texlive-fonts-extra`, but guess what, still nothing.
I stopped here. The next package would be `texlive-latex-full` (1800 MB after install and I already have a gigabyte of latex stuff sitting on my hard drive)

Clearly, satisfying these dependencies is not something you want to bother developers with. So unless I did something wrong of course, or you know a better way, PDF should definitive be turned off by default.